<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cn5X</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/XYZAB-48.svg</normaloff>images/XYZAB-48.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QGroupBox" name="grpConnexion">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>284</width>
          <height>113</height>
         </rect>
        </property>
        <property name="title">
         <string>Connexion</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Port :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cmbPort">
              <property name="minimumSize">
               <size>
                <width>158</width>
                <height>23</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Selectionnez le port de la machine grbl.</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Bauds :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cmbBauds">
              <property name="minimumSize">
               <size>
                <width>158</width>
                <height>23</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblConnectStatus">
              <property name="minimumSize">
               <size>
                <width>160</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;Non Connecté&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnConnect">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>(Dé)Connecter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>0</y>
          <width>281</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Machine status</string>
        </property>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>251</width>
           <height>197</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(224, 224, 168);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblPosX">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblPosY">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" rowspan="2">
           <widget class="QLabel" name="lblPosZ">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" rowspan="4">
           <widget class="QLabel" name="lblPosA">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLabel" name="lblPosB">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+0.000</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>LED Calculator</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>281</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QPushButton" name="btnStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/btnStart.svg</normaloff>images/btnStart.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="btnStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>150</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/btnStop.svg</normaloff>images/btnStop.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="btnUrgence">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>100</width>
           <height>100</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Double clic pour
dévérouiller l'arrêt d'urgence</string>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 25;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/btnUrgenceOff.svg</normaloff>images/btnUrgenceOff.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QPlainTextEdit" name="ptxtDebug">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>681</width>
          <height>121</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtGCode">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>340</y>
          <width>531</width>
          <height>31</height>
         </rect>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSend">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>340</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>GCode :</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnuApplication">
    <property name="title">
     <string>&amp;Application</string>
    </property>
    <addaction name="mnuAppOuvrir"/>
    <addaction name="separator"/>
    <addaction name="mnuAppQuitter"/>
   </widget>
   <addaction name="mnuApplication"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mnuAppOuvrir">
   <property name="text">
    <string>&amp;Ouvrir un fichier GCode...</string>
   </property>
  </action>
  <action name="mnuAppQuitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
